components/controls/common/km-additional-information.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <km-form-group>
+        <div class="card">
+            <div class="card-header">
+                <slot></slot>
+            </div>
+            <div class="card-body">                      
+                <km-form-group caption="" name="anyOtherInformation">
+                    <km-input-rich-lstring v-model="model" :locales="locales"></km-input-rich-lstring>
+                </km-form-group>                                    
+            </div>
+        </div>
+    </km-form-group>
+</template>
+
+<script setup lang="ts">
+    import { defineProps } from 'vue';
+
+    const model = defineModel({required:true, type:Object})
+
+    const props = defineProps({
+        locales : { type: Array<String>, required:true }
```

Comment:
<String> the only benefit of TS here.  I know here are benefits.  I will get back to it in vue.